temp_resources.tf

```diff
@@ -0,0 +1,33 @@
+// temp resources
+// use iaas for cft-api-mgmt-sbox only
+data "azurerm_subnet" "temp_subnet" {
```

Comment:
Is this just for sbox or?

Reply:
this is just used for sbox, for other env we will need to create a subnet resource rather than read iaas subnet